src/content/articles/how-i-setup-a-web-socket-client-in-nuxt.json

```diff
@@ -0,0 +1,7 @@
+{
+    "title": "How I setup a WebSocket client in Nuxt",
+    "url": "https://blog.sailscasts.com/how-i-setup-a-web-socket-client-in-nuxt",
+    "excerpt": "Sails makes it easy to setup and use WebSocket. In this article we will look at how I setup a WebSocket client to talk to a Sails API via WebSocket.",
+    "publication": "Sailscasts Blog",
+    "date": "Invalid Date"
```

Comment:
This seems to be an invalid date for some reason

Reply:
Apparently I missed, now rectified, seems that was the only error in the commit